tests/PayloadListener.Tests/Validators/EventPayloadValidatorTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Monai.Deploy.WorkflowManager.PayloadListener.Validators;
+using NUnit.Framework;
```

Comment:
I see we are using NUnit here while some other projects use xunit; do we want to discuss which one to use across all projects?